lib/jarvis/commands/merge.rb

```diff
@@ -0,0 +1,15 @@
+require "clamp"
+
+module Jarvis module Command class Merge < Clamp::Command
+
+  banner "Merge a pull request into one or more branches."
+
+  parameter "URL", "The URL to merge"
+  parameter "BRANCHES ...", "The branches to merge"
+
+  def execute
+    puts "Merging: #{url} into #{branches_list}"
```

Comment:
++ commands could be defined in their own gems if desired. For a self-contained project like Jarvis I'm not sure it's necessary, but I agree the option is there. :)